### PR TITLE
LPS-175555 Warning when a URL is duplicated outside the site

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-175555

Dear reviewer ❤️ 
Notice that the last two commits are autogenerated code, the first by the source formatted and the second because of the ant all.
Thanks!

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java [hunk 1]
```diff
@@ -3372,6 +3373,29 @@ public List<JournalArticle> getIndexableArticlesByResourcePrimKey(
 		return journalArticlePersistence.findByR_I(resourcePrimKey, true);
 	}
 
+	@Override
+	public List<Long> getJournalArticleGroupIdsByUrlTitle(
+		long companyId, String urlTitle) {
+
+		return journalArticlePersistence.dslQuery(
```
Comment (ealonso) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java:3380]: @lfbesada Why not adding this method in FriendlyURLEntryLocalizationTable?

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**